projects/plugins/jetpack/extensions/blocks/ai-assistant/inline-extensions/with-ai-extension.tsx

```diff
@@ -91,6 +92,15 @@ const blockEditWithAiComponents = createHigherOrderComponent( BlockEdit => {
 		// Jetpack AI Assistant feature functions.
 		const { increaseRequestsCount, dequeueAsyncRequest, requireUpgrade } = useAiFeature();
 
+		// Auto-scroll
+		const { snapToBottom, enableAutoScroll, disableAutoScroll } = useAutoScroll(
+			{
+				current: ownerDocument?.current?.getElementById( id ),
+			},
+			undefined,
```

Comment:
We might need to refactor the hook props to accept a single object to avoid sending `undefined`.